src/index.js

```diff
@@ -58,4 +59,25 @@ document.getElementById('table').onkeyup = (e) => {
     list[index].Description = e.target.innerHTML;
     ToLocalStorage(list);
    }
+};
+document.getElementById('table').onchange = (e) => {
+  if (e.target.className === 'todo') {
+    if (e.target.checked) {
+      e.target.parentElement.nextElementSibling.style.textDecoration = 'line-through';
+      const index = e.target.getAttribute('data-index');
+      list[index].completed = true;
+      ToLocalStorage(list);
+    } else {
+      e.target.parentElement.nextElementSibling.style.textDecoration = 'none';
+      const index = e.target.getAttribute('data-index');
+      list[index].completed = false;
+      ToLocalStorage(list);
+    }
+   }
+};
+document.getElementById('btn').onclick = () => {
+const uncompleted = list.filter((list) => !list.completed);
+list = uncompleted;
+ToLocalStorage(list);
```

Comment:
- [ ] When i click on clear all completed, the checked tasks get removed 💯 but please note that you should update all remaining items' indexes, so they represent the current list order and are unique , kindly fix it 🙏 

![INDEX](https://user-images.githubusercontent.com/89282221/215460893-42bed6bb-acf0-482b-9832-85d92e7c9d26.gif)